frontend/src/index.ts

```diff
@@ -1 +1,4 @@
+import axios from 'axios';
 import './App';
+
+axios.defaults.baseURL = process.env.BASE_URL;
```

Comment:
As I sort my testing issues,move this to the App.ts file, I've seen most configs are initialised there. Let's leave the index file as clean as we found it!.. Also test that it still works

Reply:
You will also need to add it to the docker_compose files. Check the environment section of the web service.
Then also add it to the readme as a variable that needs to be set